view/frontend/layout/algolia_search_handle.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <head>
-        <css src="Algolia_AlgoliaSearch::internals/autocomplete.css" />
-        <css src="Algolia_AlgoliaSearch::internals/grid.css" />
-        <css src="Algolia_AlgoliaSearch::internals/algolia-reset.css" />
-        <css src="Algolia_AlgoliaSearch::internals/instantsearch.v3.css" />
-        <css src="Algolia_AlgoliaSearch::internals/recommend.css" />
+        <css src="Algolia_AlgoliaSearch::js/internals/autocomplete.css" />
+        <css src="Algolia_AlgoliaSearch::js/internals/grid.css" />
```

Comment:
@damcou If we are aiming to follow [Magento's guidelines](https://developer.adobe.com/commerce/php/development/build/component-file-structure/#common-directories-1) it seems to me we should move CSS to a `css` directory. 

This is typically used for housing Less files in the `frontend` but I see straight CSS used in other places like in the `adminhtml` area in `Magento_GroupedProduct` and `Magento_TwoFactorAuth`. If we decide to switch to Less that would also be a more practical placement.